src/components/tooltip/markers/NetworkMarkerTooltipContents.js

```diff
@@ -233,6 +234,9 @@ export class NetworkMarkerTooltipContents extends React.PureComponent<Props> {
           <TooltipDetail label="Priority">
             {_getHumanReadablePriority(payload.pri)}
           </TooltipDetail>
+          <TooltipDetail label="MIME type">
+            {guessMimeTypeFromNetworkMarker(payload)}
```

Comment:
This may lie to the user, as it's the deduced type, not the actual mime type of the file. I think we should change the label to something "Guessed MIME type" or "Apparent MIME type"

Reply:
yeah good idea, we can change this back once we get the right value when https://bugzilla.mozilla.org/show_bug.cgi?id=1545069 is fixed